### PR TITLE
[GE] Bug fix for panning

### PR DESCRIPTION
**Pull request by msDestiny14**
Removed extra HTML divs that were causing the blur function to be called.

### CODE REVIEW

**Comment by msDestiny14** [comment]
Not sure where the extra code is coming from? Shouldn't this already be in master? @deltakosh

File: guiEditor/src/diagram/workbench.tsx [hunk 1]
```diff
@@ -344,11 +335,12 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
                 scene.onPointerObservable.add(panningFn, PointerEventTypes.POINTERMOVE);
                 this._panning = true;
             } else {
-                this._panning = false;
+               this._panning = false;
```
Comment (sebavan) [inline guiEditor/src/diagram/workbench.tsx:338]: NIT: looks like indentation is not the same as line 336 :-)

File: guiEditor/src/diagram/workbench.tsx [hunk 2]
```diff
@@ -199,9 +192,7 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     }
 
     componentDidMount() {
-        this._rootContainer = this.props.globalState.hostDocument.getElementById("workbench-container") as HTMLDivElement;
-        this._gridCanvas = this.props.globalState.hostDocument.getElementById("workbench-canvas-container") as HTMLDivElement;
-        this._svgCanvas = this.props.globalState.hostDocument.getElementById("workbench-svg-container") as HTMLElement;
+        this._rootContainer = this.props.globalState.hostDocument.getElementById("workbench-canvas") as HTMLDivElement;
```
Comment (sebavan) [inline guiEditor/src/diagram/workbench.tsx:195]: Why not using React Ref here as the element is part of the same component ? (won t change a lot cause there can only be one workbench per page in this case but still sounds like a good practice)

**Review by sebavan — APPROVED** [review]
Just nit comments

File: guiEditor/src/diagram/workbench.tsx (hunk 1 above)
Comment (msDestiny14) [reply guiEditor/src/diagram/workbench.tsx:338]: OH! ty!

**Pull request merged**